SpotifySearch/SpotifySearch/Album.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  Album.swift
+//  SpotifySearch
+//
+//  Created by Eric Chang on 10/28/16.
+//  Copyright © 2016 Eric Chang. All rights reserved.
+//
+
+import Foundation
+
+struct Album {
+  internal let albumName: String
+  internal let thumbnail: String
+  internal let largeImage: String
+  
+
+  static func parseData (with data: Data) -> [Album]? {
```

Comment:
I'm grading you based on your commit when submitting the exam. Yes, your issue was with parsing your items. You sort of had creating an instance of an object AND parsing keys/values intertwined. It's easier/more readable to separate the two operations. This updated code looks good though.